test/dynamo/test_functions.py

```diff
@@ -1017,14 +1017,48 @@ def test_jit_annotate(x):
         y = torch.jit.annotate(Any, x + 1)
         return y + 2
 
-    @expectedFailureDynamic
     @make_test
     def test_is_contiguous_memory_format(tensor):
         if torch.jit.is_scripting():
             return None
         elif tensor.is_contiguous(memory_format=torch.contiguous_format):
             return tensor + 1
 
+    def test_is_contiguous_frame_counts(self):
+        data = [
+            torch.rand(10),
+            torch.rand(2, 3, 32, 32).contiguous(memory_format=torch.channels_last),
+            torch.rand(10)[::2],
+            torch.rand(12),
```

Comment:
Yes, it will trigger another recompile. I had it initially but we can have 8 compilations in the cache, let me see if I can add another test case without hitting cache size limit

Reply:
You could always patch the cache_size config for the test by using the decorator
```python
@torch._dynamo.config.patch(cache_size_limit=32)
```